app/utils/user/index.ts

```diff
@@ -285,3 +285,24 @@ export function filterProfilesMatchingTerm(users: UserProfile[], term: string):
             some((suggestion) => suggestion.startsWith(trimmedTerm));
     });
 }
+
+export function getNotificationProps(user: UserModel) {
+    if (user && user.notifyProps) {
+        return user.notifyProps;
+    }
+
+    const props: Partial<UserNotifyProps> = {
+        channel: 'true',
+        comments: 'any',
+        desktop: 'all',
+        desktop_sound: 'true',
+        email: 'true',
+        first_name: (!user || !user.firstName) ? 'false' : 'true',
+        mention_keys: user ? `${user.username},@${user.username}` : '',
+        push: 'mention',
+        push_status: 'online',
+        auto_responder_active: 'false',
```

Comment:
??????

Reply:
What I meant here is that you had `auto_responder_active` but missing `auto_responder_message`, right?

Reply:
Yes right.  The component was adding both, so I removed them here. thank you .